sys/auto_init/auto_init.c

```diff
@@ -88,11 +88,18 @@
 #include "net/gcoap.h"
 #endif
 
+#ifdef FEATURE_PERIPH_HWRNG
+#include "periph/hwrng.h"
+#endif
+
 #define ENABLE_DEBUG (0)
 #include "debug.h"
 
 void auto_init(void)
 {
+#ifdef FEATURE_PERIPH_HWRNG
+    hwrng_init();
```

Comment:
Well, I put this here to have it on native, but it's actually another topic and I could just remove it from that PR